.github/workflows/deploy.yml

```diff
@@ -108,6 +123,11 @@ jobs:
 
       - name: Inspect enclave
         run: kurtosis enclave inspect ${{ env.ENCLAVE_NAME }}
+      
+      - name: List second CDK L2 chain databases (erigon)
```

Comment:
nit: name (zkevm)

Reply:
the first CDK L2 chain is using `zkevm` as the sequencer but the second one is using `erigon`